tests/system/robot/chromeTests.py

```diff
@@ -39,3 +40,40 @@ def checkbox_labelled_by_inner_element():
 		# Instead this should be spoken as:
 		"Simulate evil cat  check box  not checked"
 	)
+
+
+def test_i7562():
+	""" List should not be announced on every line of a ul in a contenteditable """
+	spy = _nvdaLib.getSpyLib()
+	_chrome.prepareChrome(
+		r"""
+			<div contenteditable="true">
+				<p>before</p>
+				<ul>
+					<li>frogs</li>
+					<li>birds</li>
+				</ul>
+				<p>after</p>
+			</div>
+		"""
+	)
+	# Tab into the contenteditable - focus mode will be enabled.
+	spy.emulateKeyPress("tab")

```

Comment:
Are we sure that we're really in focus mode at this point? Whether in focus and browse mode, the speech is the same, but the test really is meant for focus mode.
The reason I'm asking is that, when I open the content editable example from the chrome address bar, I have to disable browse mode explicitly. May be the div should also have an application role or should be nested in another div to be certain.

Reply:
Another option is to enable the speech for focus / browse mode and assert that it is spoken.